scrapely/extractors.py

```diff
@@ -16,8 +16,7 @@
 _NUMERIC_ENTITIES = re.compile("&#([0-9]+)(?:;|\s)", re.U)
 _PRICE_NUMBER_RE = re.compile('(?:^|[^a-zA-Z0-9])(\d+(?:\.\d+)?)(?:$|[^a-zA-Z0-9])')
 _NUMBER_RE = re.compile('(-?\d+(?:\.\d+)?)')
-_DECIMAL_RE = re.compile(r'(\d[\d\,]*(?:(?:\.\d+)|(?:)))', re.U | re.M)
-_VALPARTS_RE = re.compile("([\.,]?\d+)")
+_DECIMAL_RE = re.compile(r'(-?\d[\d\,\.]*(?:(?:\.\d+)|(?:)))', re.U | re.M)
```

Comment:
It looks like this regex will work if it's just:

    _DECIMAL_RE = re.compile(r'(-?\d[\d\,\.]*)', re.U | re.M)

If if needs to stay as the current regex can you add a test case for where it's needed